### DISCUSSION TITLE
1.19.2 Fabric

### DISCUSSION THREAD

**Discussion by KrekaTR** [cc-tweaked/CC-Tweaked#1816] [Q&A] [answered]
I would really like to play with this mod at Fabric 1.19.2, but I can't find such a version. This particular mod is missing from my modpack

**Comment by SquidDev** [comment, marked as answer]
For versions before 1.19.4, Fabric support is provided by [CC: Restitched](https://modrinth.com/mod/cc-restitched) instead.

**Comment by KrekaTR** [comment]
Thank you!
